### PR TITLE
Fixes #81: Cycle 2406: Add NZCH short-STARs

### LINKED ISSUE #81: Cycle 2406 SOP Changes
**NZCH**:

New short STARs are being implemented for RWY 02/20 for the mid-ILS capture. We will need to add context about how/when to use these, as well as coordination requirements with TWR/APP.

- **02**: `BELEE 1J` being implemented, and when do we use over the `6A`.
- **20**: `BELEE 1K` being implemented, and when do we use over the `6B`.

### PR DESCRIPTION
**Pull request by mwr2410**
AIRAC 2406 introduces short STARs into NZCH - `BELEE1J` & `BELEE1K` which join the `ILS DME` approaches for RWY 02/20 at the intermediate fix, shortening the approaches by 4NM.

This content covers when these arrivals should be used and how controllers can co-ordinate the use of these STARs

### CODE REVIEW

**Comment by mwr2410** [comment]
BELEE1J RUNWAY 02
NZANR 2406 BELEE1J 02

BELEE1K RUNWAY 20
NZANR 2406 BELEE1K 20

**Comment by dirtyformal** [comment]
Fixes #81 on merge.

File: docs/enroute/KAI.md [hunk 1]
```diff
@@ -90,6 +90,15 @@ KAI may clear aircraft direct to the STAR's CH TMA boundary fix without coordina
 
     **Note**: As `KABGO` sits within CH TMA's airspace, coordination would be required.
 
+#### Short STARs - NZCH
+
+`BELEE1J` and `BELEE1K` are Short STARs into CH TMA that capture ILS 02 and ILS 20 at their intermediate fixes, `IDUBU` and `DIVSU` respectively. Controllers must co-ordinate with CH TMA or CH TWR before clearing aircraft for these arrivals, as prior planning may be required to ensure flow is maintained.
```
Comment (rb-vatnz) [inline docs/enroute/KAI.md]:
```suggestion
`BELEE #J` and `BELEE #K` are Short STARs into CH TMA that capture ILS 02 and ILS 20 at their intermediate fixes, `IDUBU` and `DIVSU` respectively. Controllers must co-ordinate with CH TMA or CH TWR before clearing aircraft for these arrivals, as prior planning may be required to ensure flow is maintained.
```

File: docs/enroute/STH.md [hunk 2]
```diff
@@ -116,6 +116,14 @@ If a sequencing conflict is to occur, STH shall coordinate with CH TMA as to an
 
 STH may clear aircraft direct to the STAR's CH TMA boundary without coordination, provided that they have been cleared to rejoin the STAR thereafter. Aircraft may be cleared to track direct to a fix within CH TMA's boundary with coordination, subject to the same condition.
 
+#### Short STARs - NZCH
+
+`BELEE1J` and `BELEE1K` are Short STARs into CH TMA that capture ILS 02 and ILS 20 at their intermediate fixes, `IDUBU` and `DIVSU` respectively. Controllers must co-ordinate with CH TMA or CH TWR before clearing aircraft for these arrivals, as prior planning may be required to ensure flow is maintained.
```
Comment (rb-vatnz) [inline docs/enroute/STH.md]:
```suggestion
`BELEE #J` and `BELEE #K` are Short STARs into CH TMA that capture ILS 02 and ILS 20 at their intermediate fixes, `IDUBU` and `DIVSU` respectively. Controllers must co-ordinate with CH TMA or CH TWR before clearing aircraft for these arrivals, as prior planning may be required to ensure flow is maintained.
```

File: docs/enroute/KAI.md [hunk 3]
```diff
@@ -90,6 +90,15 @@ KAI may clear aircraft direct to the STAR's CH TMA boundary fix without coordina
...
+
+These STARs are flow control STARs, allowing controllers to sequence aircraft onto a shorter final approach. Controllers should utilise these STARs when a conflict exists at the IAF for the ILS 02/20 or if northbound or southbound arrival flows have already been established.
+
+These arrivals provide track shortening of **4NM** compared to the `A`/`B` arrivals.
```
Comment (rb-vatnz) [inline docs/enroute/KAI.md]:
```suggestion
These arrivals provide track shortening of **4nm** compared to the `BELEE #A`/`BELEE #B` arrivals.
```

File: docs/enroute/STH.md [hunk 4]
```diff
@@ -116,6 +116,14 @@ If a sequencing conflict is to occur, STH shall coordinate with CH TMA as to an
...
+
+These STARs are flow control STARs, allowing controllers to sequence aircraft onto a shorter final approach. Controllers should utilise these STARs when a conflict exists at the IAF for the ILS 02/20 or if northbound or southbound arrival flows have already been established.
+
+These arrivals provide track shortening of **4NM** compared to the `A`/`B` arrivals.
```
Comment (rb-vatnz) [inline docs/enroute/STH.md:125]:
```suggestion
These arrivals provide track shortening of **4nm** compared to the `BELEE #A`/`BELEE #B` arrivals.
```

**Review by rb-vatnz — COMMENTED** [review]
Suggestions to match style with other parts of the site

File: docs/enroute/KAI.md (hunk 3 above)
Comment (mwr2410) [reply docs/enroute/KAI.md]: Thanks Richard! Good suggestions

File: docs/enroute/KAI.md [hunk 5]
```diff
@@ -90,6 +90,15 @@ KAI may clear aircraft direct to the STAR's CH TMA boundary fix without coordina
 
     **Note**: As `KABGO` sits within CH TMA's airspace, coordination would be required.
 
+#### Short STARs - NZCH
```
Comment (dirtyformal) [inline docs/enroute/KAI.md]: I don't think mentioning of the `# K` and `# J` STARs is necessary within the KAI page, as the STAR's commencement is within the STH sector. Thoughts?

File: docs/enroute/STH.md [hunk 6]
```diff
@@ -116,6 +116,14 @@ If a sequencing conflict is to occur, STH shall coordinate with CH TMA as to an
 
 STH may clear aircraft direct to the STAR's CH TMA boundary without coordination, provided that they have been cleared to rejoin the STAR thereafter. Aircraft may be cleared to track direct to a fix within CH TMA's boundary with coordination, subject to the same condition.
 
+#### Short STARs - NZCH
+
+`BELEE #J` and `BELEE #K` are Short STARs into CH TMA that capture ILS 02 and ILS 20 at their intermediate fixes, `IDUBU` and `DIVSU` respectively. Controllers must co-ordinate with CH TMA or CH TWR before clearing aircraft for these arrivals, as prior planning may be required to ensure flow is maintained.
```
Comment (dirtyformal) [inline docs/enroute/STH.md]: Coordination would only need to occur with TMA as the primarily affected sector. TMAs can happily have aircraft intercept the midpoint without coord from the ADC.

**Review by dirtyformal — CHANGES_REQUESTED** [review]
Request small number of changes.

File: docs/enroute/KAI.md (hunk 5 above)
Comment (mwr2410) [reply docs/enroute/KAI.md]: Will move to STH

**Review by mwr2410 — COMMENTED** [review]
changes made via PR

**Review by dirtyformal — APPROVED** [review]
I'm a happy camper on my end. Send it.

**Pull request merged**